gh_actions_exporter/config.py

```diff
@@ -45,6 +45,12 @@ class Settings(BaseSettings):
     }
     flavor_label: Optional[str] = 'flavor'
     default_cost: Optional[float] = 0.008
+    github_app_private_pem: SecretStr # Checker
```

Comment:
for naming consistency with your library
```suggestion
    github_app_private_key: SecretStr # Checker
```